AppInspector/Commands/AnalyzeCommand.cs

```diff
@@ -314,8 +314,8 @@ public AnalyzeResult GetResult()
                     catch (Exception e)
                     {
                         WriteOnce.SafeLog(e.Message + "\n" + e.StackTrace, LogLevel.Error);//log details
-                        Exception f = new Exception(MsgHelp.FormatString(MsgHelp.ID.ANALYZE_FILE_TYPE_OPEN, filename));//report friendly version
-                        throw f;
+                        //Exception f = new Exception(MsgHelp.FormatString(MsgHelp.ID.ANALYZE_FILE_TYPE_OPEN, filename));//report friendly version
+                        //throw f;
                     }
```

Comment:
So this is allowing all files to be processed that will and just logging the failures rather than halt for a single failure...yeah, that's good and is actually an issue/goal.  We can take the commented code out for hygiene.